Dfe.Academies.External.Web/CypressTests/cypress/support/commands.ts

```diff
@@ -9,6 +9,24 @@ import 'cypress-file-upload';
 
 let globalApplicationId = 10080;
 
+Cypress.Commands.add('confirmApplicationDeleteVerifyElementsVisible', ():void => {
+    cy.get('.govuk-caption-xl').eq(0).then(($applicationId)=> {
+        const applicationId = $applicationId.text().split('_').pop().replace(/\s/g, '')
+        cy.log(`Application ID = ${applicationId}`)
+        globalApplicationId = Number(applicationId)
+        cy.log(`Global Application ID = ${globalApplicationId}`)
+
+        cy.get('.govuk-back-link').contains('Back')
+        cy.get('.govuk-caption-xl').contains('Application reference: ')
+        cy.get('.govuk-heading-l').contains('Are you sure you want to delete this application?')
+        cy.get('.govuk-warning-text').contains('This action cannot be undone')
+        cy.get('.govuk-body').contains('It will be deleted for all contributors and any information that has been given will be lost. Any schools on this application that want to become academies later must start a new application.')
+        
```

Comment:
perhaps  remove all the contains text stuff ..  but  add on contains ('''Application ref :{applicationReference}).

verifying the displayed application ref is correct...

and id maybe have it in the is method

 A2BYourApplication.selectCancelApplication(); 

which would contain
verify appliction ref is correct
click  cancel button

doing away with  A2BConfirmApplicationDelete.confirmApplicationDeleteVerifyElementsVisible(); all together